scripts/shared/e2e.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+## Process command line flags ##
+
+source /usr/share/shflags/shflags
+DEFINE_string 'cluster_settings' '' "Settings file to customize cluster deployments"
+DEFINE_string 'focus' '.*' 'Ginkgo focus for the E2E tests'
+FLAGS "$@" || exit $?
+eval set -- "${FLAGS_ARGV}"
+
+focus="${FLAGS_focus}"
+cluster_settings="${FLAGS_cluster_settings}"
+
+set -o pipefail 
+set -em
```

Comment:
Is there some reason I'm missing to do this `set` in two lines?

```suggestion
set -em -o pipefail
```

Reply:
Thanks i was thinking about it at some point but forgot to merge em, i've added the commit to the PR :)